[FSTORE-360][Append] Small fixes, added pypi link, revised OSX/Windows instructions (#139)
logicalclocks/logicalclocks.github.io: docs/user_guides/client_installation/index.md

```
pip install hopsworks
```
Supported versions of Python: 3.7, 3.8, 3.9 ([PyPI ↗](https://pypi.org/project/hopsworks/))

!!! attention "OSX Installation"

>     On OSX systems you need to install librdkafka manually before installing hopsworks. You can do so by running the following commands:

Reviewer: ```suggestion
    On OSX systems you need to install librdkafka manually before installing hopsworks. You can verify if you have installed it previously using `brew info librdkafka`. If it is not installed yet, you can do so using `brew install`, however, you always need to set the `C_INCLUDE_PATH` and `LIBRARY_PATH`:
```

>     ```
>     brew install librdkafka

Reviewer: ```suggestion
    If not installed yet, install librdkafka:
    ```
    brew install librdkafka
    ```
    If it is already installed, set the environment variables and proceed with installing the hopsworks client library:
    ```
```

```
pip install hsfs[python]
```
Supported versions of Python: 3.7, 3.8, 3.9 ([PyPI ↗](https://pypi.org/project/hsfs/))

!!! attention "OSX Installation"

>     On OSX systems you need to install librdkafka manually before installing hsfs. You can do so by running the following commands:

Reviewer: ```suggestion
    On OSX systems you need to install librdkafka manually before installing hopsworks. You can verify if you have installed it previously using `brew info librdkafka`. If it is not installed yet, you can do so using `brew install`, however, you always need to set the `C_INCLUDE_PATH` and `LIBRARY_PATH`
```

>     ```
>     brew install librdkafka

Reviewer: ```suggestion
    If not installed yet, install librdkafka:
    ```
    brew install librdkafka
    ```
    If it is already installed, set the environment variables and proceed with installing the hopsworks client library:
    ```
```

```
pip install hsfs[python]
```
Supported versions of Python: 3.7, 3.8, 3.9 ([PyPI ↗](https://pypi.org/project/hsfs/))

!!! attention "OSX Installation"

>     On OSX systems you need to install librdkafka manually before installing hopsworks. You can verify if you have installed it previously using `brew info librdkafka`. If it is not installed yet, you can do so using `brew install`, however, you always need to set the `C_INCLUDE_PATH` and `LIBRARY_PATH`

Reviewer: ```suggestion
    On OSX systems you need to install librdkafka manually before installing hopsworks. You can verify if you have installed it previously using `brew info librdkafka`. If it is not installed yet, you can do so using `brew install`, however, you always need to set the `C_INCLUDE_PATH` and `LIBRARY_PATH`.
```
